tests/extension/oneapi_non_uniform_groups/group_barrier.h

```diff
@@ -148,19 +148,21 @@ struct non_uniform_group_barrier_test {
                   "memory_scope fence_scope) is wrong\n");
 
               // test of default barrier
-              local_acc[llid] = llid;
-              sycl::group_barrier(non_uniform_group);
+              if (sub_group.get_group_linear_id() == 0){
```

Comment:
Tagging @steffenlarsen for feedback here. Would it be a better fix (in terms of coverage) to make the logic around `local_acc` more sophisticated to account for multiple sub-groups doing those calculations? I.e. we offset all indices that we use by `sub_group.get_group_id() * sub_group.get_max_local_range().size()` to make sure that every sub-group operates on a different section of the local accessor (adjusting the local accessor size accordingly to accommodate more data)

Reply:
Agreed! Much better. ⭐